can't connect to RethinkDB in production when running PFA

```
Oct 28 17:14:59 game app/cycleLaunchedWorker.1:  Creating a pool connected to aws-us-east-1-portal.15.dblayer.com:11204 
Oct 28 17:16:14 game app/cycleLaunchedWorker.1:  Forming teams for cycle 17 of chapter 0edb08e1-d8ab-4318-8363-0711a7f9edbb 
Oct 28 17:16:36 game app/cycleLaunchedWorker.1:  Entering slow growth mode 
Oct 28 17:16:36 game app/cycleLaunchedWorker.1:  Fail to create a new connection for the connection pool. Error:{"msg":"Failed to connect to aws-us-east-1-portal.15.dblayer.com:11204 in less than 20s","message":"Failed to connect to aws-us-east-1-portal.15.dblayer.com:11204 in less than 20s.","isOperational":true} 
< ... REPEATED 300+ TIMES ... >
Oct 28 17:16:36 game app/cycleLaunchedWorker.1:  Fail to create a new connection for the connection pool. Error:{"msg":"Failed to connect to aws-us-east-1-portal.15.dblayer.com:11204 in less than 20s","message":"Failed to connect to aws-us-east-1-portal.15.dblayer.com:11204 in less than 20s.","isOperational":true} 
Oct 28 17:16:38 game heroku/cycleLaunchedWorker.1:  State changed from up to crashed 
Oct 28 17:16:38 game heroku/cycleLaunchedWorker.1:  Process exited with status 1 
Oct 28 17:16:38 game heroku/cycleLaunchedWorker.1:  State changed from crashed to starting 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  events.js:154 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:        throw er; // Unhandled 'error' event 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:        ^ 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  Error: write after end 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at writeAfterEnd (_stream_writable.js:159:12) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at TLSSocket.Writable.write (_stream_writable.js:204:5) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at TLSSocket.Socket.write (net.js:620:40) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at Connection._sendProof (/app/node_modules/rethinkdbdash/lib/connection.js:281:19) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at Connection._computeSaltedPassword (/app/node_modules/rethinkdbdash/lib/connection.js:247:10) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at TLSSocket.<anonymous> (/app/node_modules/rethinkdbdash/lib/connection.js:184:18) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at emitOne (events.js:90:13) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at TLSSocket.emit (events.js:182:7) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at readableAddChunk (_stream_readable.js:147:16) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at TLSSocket.Readable.push (_stream_readable.js:111:10) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:      at TLSWrap.onread (net.js:525:20) 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Linux 3.13.0-100-generic 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "run" "workers:cycleLaunched" 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! node v5.6.0 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! npm  v3.7.5 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! code ELIFECYCLE 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! game@0.1.4 workers:cycleLaunched: `npm run symlinks && ./node_modules/.bin/babel-node ./server/workers/cycleLaunched` 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Exit status 1 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR!  
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Failed at the game@0.1.4 workers:cycleLaunched script 'npm run symlinks && ./node_modules/.bin/babel-node ./server/workers/cycleLaunched'. 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Make sure you have the latest version of node.js and npm installed. 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! If you do, this is most likely a problem with the game package, 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! not with npm itself. 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Tell the author that this fails on your system: 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR!     npm run symlinks && ./node_modules/.bin/babel-node ./server/workers/cycleLaunched 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! You can get information on how to open an issue for this project with: 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR!     npm bugs game 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Or if that isn't available, you can get their info via: 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR!     npm owner ls game 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! There is likely additional logging output above. 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR! Please include the following file with any support request: 
Oct 28 17:16:38 game app/cycleLaunchedWorker.1:  npm ERR!     /app/npm-debug.log 
```

--- a/server/actions/formProjects.js
+++ b/server/actions/formProjects.js
@@ -4,6 +4,7 @@
  * to a project team that will work on their most-preferred goal.
  */
 
+import Promise from 'bluebird'
 import {getCycleById} from 'src/server/db/cycle'
 import {findPlayersByIds} from 'src/server/db/player'
 import {findVotesForCycle} from 'src/server/db/vote'
@@ -122,20 +123,22 @@ async function _buildVotingPool(cycleId) {
 }
 
 async function _getPlayerFeedback(playerIds) {
-  const feedback = {respondentIds: {}}
+  const pairings = flatten(playerIds.map(respondentId => {
+    const teammates = playerIds.filter(id => id !== respondentId)
+    return teammates.map(subjectId => ({respondentId, subjectId}))
+  }))
 
-  await Promise.all(
-    playerIds.map(respondentId => {
-      feedback.respondentIds[respondentId] = {subjectIds: {}}
-      const teammates = playerIds.filter(id => id !== respondentId)
-      return Promise.all(teammates.map(subjectId =>
-        getLatestFeedbackStats({respondentId, subjectId})
-          .then(stats => {
-            feedback.respondentIds[respondentId].subjectIds[subjectId] = stats
-          })
-      ))
-    })
+  const feedbackTuples = await Promise.map(
+    pairings,
+    pair => getLatestFeedbackStats(pair).then(stats => ({...pair, stats})),
+    {concurrency: 20}
   )
+
+  const feedback = feedbackTuples.reduce((result, {respondentId, subjectId, stats}) => {
+    result.respondentIds[respondentId] = result.respondentIds[respondentId] || {subjectIds: {}}
+    result.respondentIds[respondentId].subjectIds[subjectId] = stats
+    return result
+  }, {respondentIds: {}})
 
   return feedback
 }

